content/SCALE/SCALETutorials/Storage/Pools/PermissionsSCALE.md

```diff
@@ -19,112 +19,231 @@ ACL permissions control the actions users can perform on dataset contents.
 
-Click **Save** if you do not want to use an ACL preset.
+If you want to change the **ACL Type** and **ACL Mode** settings, do it before you begin using the dataset in your production environments.
 
-### Configuring an ACL Preset (NFSv4 ACL)
 {{< hint warning >}}
-WARNING: Changing the ACL type affects how TrueNAS writes and reads on-disk ZFS ACL.
+WARNING: Changing the **ACL Type** affects how TrueNAS writes and reads on-disk ZFS ACL.
 
 When the ACL type changes from POSIX to NFSv4, internal ZFS ACLs do not migrate by default, and access ACLs encoded in posix1e extended attributes convert to native ZFS ACLs. 
 
-When the ACL type changes from NFSv4 to POSIX, native ZFS ACLs do not convert to posix1e extended attributes, but ZFS will use the native ACL for access checks.    
+When the ACL type changes from NFSv4 to POSIX, native ZFS ACLs do not convert to posix1e extended attributes, but ZFS uses the native ACL for access checks.    
 
 To prevent unexpected permissions behavior, you must manually set new dataset ACLs recursively after changing the ACL type.   
 
 Setting new ACLs recursively is destructive. We suggest creating a ZFS snapshot of the dataset before changing the ACL type or modifying permissions.
 {{< /hint >}}
+{{< expand "Click Here for More Information" "v" >}}
+From the **Datasets** screen:
+
+To change the ACL settings on a new dataset, click **Add Dataset**, then click **Advanced Options** and scroll down to the **ACL Type** and **ACL Mode** settings. 
+To change the ACL settings on an existing dataset, click **Edit** on the **Dataset Details** widget to open the **Edit Dataset** screen.  
+Click **Advanced Options**, then scroll down to the **ACL Type** and **ACL Mode** settings. 
+
+![EditDatasetAdvancedACLOptions](/images/SCALE/22.12/EditDatasetAdvancedACLOptions.png "Edit Dataset Advanced Options for ACLs")
+
+You cannot change the **ACL Mode** setting until you select the **ACL Type** from the dropdown list. 
+This changes the **ACL Mode** setting to the setting that works for the selected type. 
+
+Changing the **ACL Type** from **Inherit** to any other type displays a warning window with the impact of the change.
+
+![ChangeACLTypeWarnings](/images/SCALE/22.12/ChangeACLTypeWarning.png "Changing ACL Type Warning")
+
+Leave **ACL Type** set to **Inherit** to preserve the ACL type from the parent dataset. For SCALE, which is based on Linux, select either **SMB/NFSv4** or **POSIX** options. 
+
+NFSv4 is richer than POSIX and is used to losslessly migrate Windows-style ACLs across Active Directory domains (or stand-alone servers). 
+POSIX ACLs are a Linux-specific ZFS feature, used when an organization data backup target does not support native NFSv4 ACLs. 
+Since the Linux platform used POSIX for a long time, many backup products that access the server outside the SMB protocol cannot understand or preserve native NFSv4 ACLs. 
```

Comment:
```suggestion
NFSv4 is richer than POSIX and is used to losslessly migrate Windows-style ACLs across Active Directory domains (or stand-alone servers). 
POSIX ACLs are a Linux-specific ZFS feature used when an organization data backup target does not support native NFSv4 ACLs. 
Since the Linux platform used POSIX for a long time, many backup products that access the server outside the SMB protocol cannot understand or preserve native NFSv4 ACLs. 
```